### PR TITLE
Rarespopa branch

### PR DESCRIPTION
**Pull request by popararesd**
Added Rares Popa Tasks

### CODE REVIEW

File: rares-popa-dir/rarespopa.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Rares Popa  created this file during Lab 5
+second commit
```
Comment (raduchistest) [inline rares-popa-dir/rarespopa.txt:2]: Add reviewed by RaduChis

**Review by raduchistest — CHANGES_REQUESTED** [review]

File: rares-popa-dir/rarespopa.txt (hunk 1 above)
Comment (popararesd) [reply rares-popa-dir/rarespopa.txt:2]: Done

**Review by BogdanCanciuACS — CHANGES_REQUESTED** [review]
Add reviewed by bogdanCanciu

**Review by BogdanCanciuACS — APPROVED** [review]

**Comment by BogdanBadiu** [comment]
Please add "Reviewed by Bogdan-Vasile Badiu"

**Pull request merged**